docs/auth.md

```diff
@@ -43,26 +233,23 @@ aggregates them into their respective files in `$HOME`.
        tekton.dev/git-0: github.com # Described below
    type: kubernetes.io/ssh-auth
    data:
-     ssh-privatekey: <base64 encoded>
+     ssh-privatekey: <base64-encoded>
      # This is non-standard, but its use is encouraged to make this more secure.
      # If it is not provided then the git server's public key will be requested
      # when the repo is first fetched.
-     known_hosts: <base64 encoded>
+     known_hosts: <base64-encoded>
    ```
 
-   `tekton.dev/git-0` in the example above specifies which web address these
-   credentials belong to. See
-   [Guiding Credential Selection](#guiding-credential-selection) below for more
-   information.
+   In the above example, the value for `tekton.dev/git-0` specifies the URL for which Tekton will use this `Secret`,
+   as described in [Understanding credential selection](#understanding-credential-selection).
+
+1. Generate the `ssh-privatekey` value. For example:
 
-1. Generate the value of `ssh-privatekey` by copying the value of (for example)
-   `cat ~/.ssh/id_rsa | base64`.
+   `cat ~/.ssh/id_rsa | base64`
```

Comment:
See above about stringData and dropping the need to base64 encode

Reply:
Which lines should I change here and to what?

Reply:
I'll try to comment on the lines.